approval_polls/templates/approval_polls/edit.html

```diff
@@ -118,16 +120,20 @@ <h4 class='panel-title'>
               </div>
               <div class='radio'>
                 <label class='full-width'><input type='radio' name='radio-poll-type' {% if poll.vtype == 3 %} checked='true' {% endif %} value='3'>Only specific Email owners (will be invited by Email).
+                  <div id='existing-emails' hidden>
+                    <h6>List of current email invitees(You can copy either of these and paste into the field below):</h6>
+                    <h6>{{ invited_emails|default:'No current invitees' }}</h6>
+                  </div>
                   <div id='email-input' hidden>
-                    <h6>Enter comma-separated Email IDs:</h6>
+                    <h6>(Enter a comma after each email ID:)</h6>
                     <input type='text' name='token-emails' class='form-control' id='tokenEmailField' placeholder='Email IDs' />
                     <label id='email-error' for='token-emails' class='alert alert-danger full-width' hidden>One or more Email addresses are invalid.
                     </label>  
                   </div>
                 </label>
               </div>
               <hr>
--->
+
               <p>Poll Visibility</p>
               <div class='checkbox'>
                 <label><input type='checkbox' id='poll-visibility' name='public-poll-visibility' value='' autocomplete='off' {% if poll.is_private %}{% else %}checked{% endif %}>List this poll on the home page.</label>
```

Comment:
Currently for poll type 3, this option seems to be unchecked by default. If the poll is forced to be displayed on the home page even if it has email invitees, the edit page should accordingly display this option as checked.